### PR TITLE
NEV Port - Consciousness Backup Implant

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

**Tl;dr - ports NEV's Soulcrypts and renames them to Consciousness Backup Implants.** 

This PR is to go hand-in-hand with the 'nerfing defibs' PR that reduces the time allowed to revive someone using a defib post-death. The Con-Back implant effectively is an implant that can be purchased from Medical, using their new implant fabricator machine to make extras, that allows revival via cloning post 20 minute timer death.

This implant is incompatible with psionics, cruciform, and death alarm implants. Effectively being an alternative to those options. It contains a handful of downsides, such as higher nutrition usage and EMP-susceptibility, but its main difficulty is its price. Costing 750 credits base to buy unless medical sells them cheaper. This is due to these implants being a 'one-time-buy' of working the entire round until broken.

These implants will receive further development as time goes but currently it's intended to make death sport consequences with or without this implant; making you buy into the ability to be revived if you are not quick to do so - and still suffering perma stat loss if cloned in such a way.
	
<hr>
</details>

## Changelog
:cl:
adds: Consciousness backup implant
adds: New implant fabricator for CAPSA capable of making death alarms, psionic organs (for now), and consciousness backup implants.
fixes: Cloning tanks and cloning readers now work.
adds: New cloning reader sprite made by me using the old disk-cloner as a base. It's even animated now!!!
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
I believe in Hestia/Vesta/Whatever this was called a...Neural Lace, I believe? The implant that you could opt on character setup that allowed your body to be cloned since it "stored all the information about yourself", the process was to take that off the dead body, slap the corpse on the scanner, clone a new body, insert the Neural Lace on the new clone's head, then defib back to life. Will this be the same process? I need to know to start writing lore behind this.

File: code/modules/core_implant/cruciform/machinery/cloning.dm [hunk 1]
```diff
@@ -406,23 +405,28 @@
 /////////////////////
 
 /obj/machinery/neotheology/reader
-	name = "strange scanner"
-	desc = "It thrums and seems to be scanning anyone who gets near it judging from the small beeps."
+	name = "Conciousness Implant Reader"
+	desc = "A strangely advanced machine that scans the user for any signs existing conciousness through brainwaves. \
```
Comment (ThePainkiller) [inline code/modules/core_implant/cruciform/machinery/cloning.dm:409]:
```suggestion
	desc = "A strangely advanced machine that scans the user for any signs of existing conciousness through magnetic resonance imaging. \
```

File: code/modules/core_implant/cruciform/machinery/cloning.dm [hunk 2]
```diff
@@ -406,23 +405,28 @@
...
+			Used to check if a user's conciousness backup implant has remained intact; allowing the transfer of conciousness to a new body."
```
Comment (ThePainkiller) [inline code/modules/core_implant/cruciform/machinery/cloning.dm:410]:
```suggestion
			Used to check if a user's conciousness backup implant has remained intact, allowing the transfer of conciousness to a new body."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

**Comment by Stutternov** [comment]
> I believe in Hestia/Vesta/Whatever this was called a...Neural Lace, I believe? The implant that you could opt on character setup that allowed your body to be cloned since it "stored all the information about yourself", the process was to take that off the dead body, slap the corpse on the scanner, clone a new body, insert the Neural Lace on the new clone's head, then defib back to life. Will this be the same process? I need to know to start writing lore behind this.

Right, so - overall this device is designed to do something semi-similar lore wise.

The general idea is that it holds an engram of your consciousness and DNA; so even after you die your consciousness will be backed up onto the device. Effectively leaving your body 'dead' but saving what would be determined as your consciousness into the device; allowing a 'life-after-death' effectively.

This will **NOT** be for setup upon decision; this will instead be an implant type CAPSA can make with a new implant fabricator they'll be getting (allows construction of psionic implants, death alarms, and Con-Backs for now). This will effectively be a 'trade-off' implant, disallowing the usage of psionics and death alarms but allowing you, if you have enough energy loaded in the thing, to be revived post-death.

The reason these will not be in setup lore wise is a 'low shelf life' or something to that effect. The backup only realistically lasting for so long.
